### DISCUSSION TITLE
对“不重复使用标点符号”的反对意见

### DISCUSSION THREAD

**Discussion by HeavySnowJakarta** [sparanoid/chinese-copywriting-guidelines#165] [General]
据 GB/T15834-2011 标点符号用法标准，对叠用感叹号的规定如下：

> 4.2.3.3在多个问句连用或表达疑问语气加重时，可叠用问号。通常应先单用，再叠用，最多叠用三个问号。在没有异常强烈的情感表达需要时不宜叠用问号。
示例：这就是你的做法吗？你这个总经理是怎么当的？？你怎么竟敢这样欺骗消费者？？？
> 4.3.3.3表示声音巨大或声音不断加大时，可叠用叹号；表达强烈语气时，也可叠用叹号，最多叠用三个叹号。在没有异常强烈的情感表达需要时不宜叠用叹号。
示例1：轰！！在这天崩地塌的声音中，女娲猛然醒来。
示例2：我要揭露！我要控诉！！我要以死抗争！！！
> 4.3.3.4当句子包含疑问、感叹两种语气且都比较强烈时（如带有强烈感情的反问句和带有惊愕语气的疑问句），可在问号后再加叹号（问号、叹号各一）。
示例1:这么点困难就能把我们吓倒吗？！
示例2:他连这些最起码的常识都不懂，还敢说自己是高科技人材？！

故我认为，“不重复使用标点符号”这一规定并不合适。

如果这里不是此约定的 original place of issue，请告诉我 Where is it，十分感谢！

**Comment by liyishuai** [comment]
Duplicate of #32

**Comment by sparanoid** [comment]
本项目的初衷只是为了「文案排版」更加美观好看，并在团队内部形成统一的规范，因此对一些标点的标准用法会有出入
